### PR TITLE
Selenium: Refactor some tests from the 'editor' package

### PR DESCRIPTION
**Pull request by artaleks9**
### What does this PR do?
* Does refactor some selenium tests from the _editor_ package
* Related selenium tests: _AutocompleteProposalJavaDocTest_, _SplitEditorFeatureTest_ 

### What issues does this PR fix or reference?
#12312

### CODE REVIEW

**Comment by artaleks9** [comment]
ci-test

**Review by SkorikSergey — APPROVED** [review]

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/editor/autocomplete/AutocompleteProposalJavaDocTest.java [hunk 1]
```diff
@@ -139,11 +111,11 @@ public void shouldDisplayJavaDocOfClassMethod() throws Exception {
         "concat(String part1, String part2, char divider) : String App");
 
     // then
-    checkProposalDocumentationHTML(expectedJavadocHtmlText);
+    checkProposalDocumentationHTML(getJavadocHtmlText());
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/editor/autocomplete/AutocompleteProposalJavaDocTest.java]: `getExpectedJavadocHtmlText()` method name is more relevant to method intention, IMHO.

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/editor/SplitEditorFeatureTest.java [hunk 2]
```diff
@@ -29,86 +30,84 @@
...
-  private static final String NAME_JAVA_CLASS = "GreetingController";
+  private static final String TEXT_FILE = "README.md";
+  private static final String PATH_TEXT_FILE = PROJECT_NAME + "/" + TEXT_FILE;
   private static final String NEW_NAME = "NewName";
   private static final String NEW_NAME_JAVA = "NewName.java";
   private static final String NEW_NAME_TXT_FILE = "NewREADME";
-
   private static final String TEXT = "some text";
 
+  private final String tabNameJavaFile = getTabNameJavaFile();
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/editor/SplitEditorFeatureTest.java]: `private final String javaFileName = getJavaFileNameFromTabTitle();` variable and method names looks more obviously, IMHO.

**Comment by artaleks9** [comment]
ci-test

**Review by dmytro-ndp — APPROVED** [review]

**Comment by artaleks9** [comment]
ci-test

**Pull request merged**